plugins/generator-1.16.5/forge-1.16.5/mappings/villagerprofessions.yaml

```diff
@@ -0,0 +1,16 @@
+_default: VillagerProfession.ARMORER
```

Comment:
Default should be NITWIT as it's the "No profession" villager

Reply:
I'll remove NITWIT to prevent anyone to use it in the trade mod element (because NIWIT can't have trades)